Kickstarter-iOS/DataSources/SettingsAccountDataSource.swift

```diff
@@ -11,7 +11,7 @@ final class SettingsAccountDataSource: ValueCellDataSource {
 
     self.filteredSections = shouldHideEmailPasswordSection
       ? SettingsAccountSectionType.allCases.filter { $0 != .emailPassword }
```

Comment:
Wonder if we should change `emailPassword` case to something more descriptive (just like `createPassword`)..i.e. `changeEmailPassword`?